k8s/demo/common/ccd/latest-ccd-chart.yaml

```diff
@@ -21,6 +21,27 @@ spec:
   values:
     tags:
       idam-pr: true
+
+    ccd:
+      emAnnotation:
+        enabled: true
+      ccpay:
+        enabled: true
+      draftStore:
+        enabled: true
+      dmStore:
+        enabled: true
+      managementWeb:
+        enabled: true
+      apiGatewayWeb:
+        enabled: true
+      activityApi:
+        enabled: true
+      blobstorage:
```

Comment:
when you enable `blobstorage` there is a property of def store you have to set to true so that we can see the list of imports in Admin Web: AZURE_STORAGE_DEFINITION_UPLOAD_ENABLED.
Please ask Ola to verify that this worked and we can see the list of imports on Admin Web